staged_dependencies.yaml

```diff
@@ -15,24 +15,12 @@ downstream_repos:
   insightsengineering/goshawk:
     repo: insightsengineering/goshawk
     host: https://github.com
-  insightsengineering/teal:
-    repo: insightsengineering/teal
-    host: https://github.com
   insightsengineering/teal.data:
     repo: insightsengineering/teal.data
     host: https://github.com
```

Comment:
There is a task assigned for https://github.com/insightsengineering/teal.data/issues/145, which involves removing scda from teal.data. I'm a little uncertain about whether this removal should be done now or at a later time. Should pull request be  kept it on hold until the task is merged?

Reply:
We can merge what is ready now and open `decouple_scda@main` again for other PRs. It's not a problem.